config/horizon.json

```diff
@@ -80,6 +80,7 @@
 		"plugins/ssr-categories": false,
 		"plugins/ssr-landing": true,
 		"post-editor/checkout-overlay": true,
+		"post-list/qr-code-link": false,
```

Comment:
I was thinking about it, but then I thought: wouldn't it be better to set it as false as a way of saying "hey, we introduced a new feature and if you wanted here, you need to set it true!". We are not including it (even on false) on jetpack-cloud environments because that feature actually doesn't work there, but in case of Calypso Blue I think it would be okay to declare it at least somehow. What do you think?

Reply:
I see your point of view, but I really see all those false flags as unnecessary noise. If they have no real purpose, then they should be removed IMHO. I don't feel so strongly about it, though, so I'll leave it to your judgement.